metadata.rb

```diff
@@ -46,7 +46,7 @@
 supports 'scientific'
 supports 'ubuntu'
 
-depends 'yum-epel', '~> 0.5'
+depends 'yum-epel', '> 0.5'
```

Comment:
Even if we update the version number, I think we should conserve the pessimistic operator, shouldn't we?